.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,58 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server
+    We appreciate your time and interest in helping our project!
+
+    Use this template to help us review your change. Not everything is required,
+    depending on your change. Keep or delete what is relevant for your change.
+    Remember that it helps us review if you give more helpful info for us to
+    understand your change.
+
+    Ensure that you’ve read through and followed the Contributing Guidelines, in
+    https://github.com/metabrainz/musicbrainz-server/blob/master/CONTRIBUTING.md
+-->
+
+# Summary
+<!--
+    Just give us a overview of your change, don’t go into technical details yet.
+-->
+
+:beginner: Describe this change in 1-2 sentences
```

Comment:
If the change is minor, this section probably isn't useful at all, just rephrasing things from the problem/solution sections which are likely only 1 sentence each. I know the template says you can remove irrelevant parts, but it doesn't really say when they might be irrelevant, so perhaps we can just say "You can remove this section if your change is small enough that the problem and solution sections don't need a summary."

Reply:
If the change is really minor, no section is useful, the whole template can be erased and replaced by a single sentence. It doesn’t seem useful to repeat it for every part. Maybe it can be clarified in the introductory comment without lengthening it too much? It currently says:

> Not everything is required, depending on your change. Keep or delete what is relevant for your change.

Reply:
If I were a new contributor I think I'd be very compelled to fill out the whole template even for a minor change, because it makes the process look very formalized (and I wouldn't want to play fast and loose with any rules the org has).

So even just paraphrasing what you said here would be useful, I think

> Not everything is required, depending on your change. Keep or delete what is relevant; if your change is fairly minor, feel free to disregard the template, as it's only a guideline.

Reply:
Or the comment of this section can be clarified as follows?
> Just give us a overview of your change <ins>if needed</ins>, don’t go into technical details yet.

Or the section can be just dropped? It used to hold the type of change (bug fix, feature addition, ...) but it isn’t here. Plus, I just noticed description is never filled in [PRs to Picard](https://github.com/metabrainz/picard/pulls).

Reply:
I replied without seeing your reply first, but we’re looking in the same direction.

Reply:
Because paraphrasing was making the text much more longer, I rephrased and split the whole paragraph in https://github.com/metabrainz/musicbrainz-server/commit/410198fd412dd45e62862f79effbc293d55fc4c7#diff-195a635ad245ded9076330e31134bd80R3-R8 so as to highlight that unneeded parts for the current change should be removed.

Reply:
Finally dropped with https://github.com/metabrainz/musicbrainz-server/commit/1d1bda4deed9ff96af16123c394449dd4cc456ff after https://github.com/metabrainz/musicbrainz-server/pull/1345#discussion_r409311523.